### DISCUSSION TITLE
Load Model with the new Openai API

### DISCUSSION THREAD

**Discussion by forpr1093** [oobabooga/text-generation-webui#4896] [Q&A] [answered]
May I know how to complex load model using the new Openai API? I was using the base API to load model through API, but it has been removed and I couldn't find any example out there to load model using the new Openai API.

**Comment by michusx** [comment, marked as answer]
Hi.
Check SWAGER UI webpage for your instance of OpenAPI running at: 
http://127.0.0.1:5000/docs (or your serwer IP address) 
to find out all available methods including loading model.

**Reply by forpr1093** [reply]
Thank you!
